app/js/components/validation.js

```diff
@@ -25,9 +25,10 @@ if (form.length) {
         const parent = $(this).closest('.form-row');
         const errorContainer = parent.parents('.form-row').find('.parsley-errors').first();
         const translatedValidationText = $(this).data('motivation-keep-talking');
-        const words = $(this).val().split(' ').filter(v => v !== "");
+        const val = $(this).val();
+        const words = val.split(' ').filter(v => v !== "");
 
-        if (words.length < 10) {
+        if (val !== '' && words.length < 10) {
```

Comment:
Just nitpicking here.. Does jQuery val also trim the input of whitespaces? Using `words.length > 0 && words.length < 10` might be a bit better expression?